indonesia flag

skmmks opened the pull request:
<img width="1000" alt="Screen Shot 2019-07-06 at 3 43 00 PM" src="https://user-images.githubusercontent.com/47374317/60761776-e2fc2700-a004-11e9-95ac-d2b2afd6147e.png">

skmmks committed:
indonesia flag

dpaschal-lf commented on flags.css:
```diff
@@ -1,5 +1,14 @@
 /* Indonesia */
-
+.flagContainer {
+    width: 50vw;
```
you might want to make them based on vmin that way the flag remains proportional.

like
```
height: 50vmin;
width: 33vmin;
```

dpaschal-lf commented on flags.css:
```diff
@@ -1,5 +1,14 @@
+.flagContainer {
+    width: 50vw;
+    height: 50vh;
+    border: solid black;
```
huh, does it do 1px by default?

dpaschal-lf commented on flags.css:
```diff
@@ -1,5 +1,14 @@
+    height: 50vh;
+    border: solid black;
+}
+.red {
```
since this is more than just red, I recommend making the name more expressive as to what it is.  It is a half height element.  And it is red.  Maybe it is a indonesiaRedPanel?

Also you don't need width 100%, because is that by default

skmmks committed:
change view properties to vmin
change css class name
remove width on css rule
change classname on indonesia div

dpaschal-lf commented on flags.html:
```diff
@@ -5,79 +5,79 @@
+    <div class="indonesiaRed"></div>
 </div>
-<div id="columbia" class="flagContainer">
+<!--<div id="columbia" class="flagContainer">-->
```
why comment out everything else?

skmmks replied:
I wanted to have just one div light up at a time on my IDE.  
<img width="593" alt="Screen Shot 2019-07-06 at 6 13 11 PM" src="https://user-images.githubusercontent.com/47374317/60762629-c7e7e200-a019-11e9-930d-afe3a713bdbc.png">

dpaschal-lf commented on flags.html:
```diff
@@ -5,79 +5,79 @@
     <body>
 <div id="indonesia" class="flagContainer">
-
+    <div class="indonesiaRed"></div>
```
this could be dangerous.  basically you are relying on your background to be white.  That's a bad idea.  Please fix this.  If the background turned a different color, your element's background isn't white, it's transparent.

dpaschal-lf reviewed (commented):
this code was not fully reviewed.  Please don't merge branches before they are reviewed or else you'll find yourself super fired.
